cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -233,7 +277,12 @@ private boolean setSchemaCompatibility(
       Map<String, String> params = new HashMap<>();
       if (isForce) {
         params.put("compatibility", "NONE");
-      } else params.put("compatibility", defaultSchemaCompatibility);
+      } else if (schemaCompatibilityMode != null && !schemaCompatibilityMode.isBlank()) {
+        // Allows us to specify the schema compatibility to the exact type we want.
+        params.put("compatibility", schemaCompatibilityMode);
+      } else {
+        params.put("compatibility", defaultSchemaCompatibility);
```

Comment:
Just realizing, we should also introduce global schema compatibility. May be in another PR